tests/integration/targets/aci_l3out_extsubnet/tasks/main.yml

```diff
@@ -1,5 +1,5 @@
 # Test code for the ACI modules
-# Copyright: (c) 2020, Cindy Zhao (@cizhao)
+# Copyright: (c) 2020, Anvitha Jain (@anvitha-jain)
```

Comment:
Is this file still needed? Seems the "tests/integration/targets/aci_l3out_static_routes/tasks/main.yml" is the correct one.

Reply:
tests/integration/targets/aci_l3out_extsubnet/tasks/main.yml  is the correct one.

"tests/integration/targets/aci_l3out_static_routes/tasks/main.yml" - is the correct one